src/ebur128.rs

```diff
@@ -935,6 +1002,16 @@ mod tests {
     #[cfg(feature = "c-tests")]
     use quickcheck_macros::quickcheck;
 
+    fn f64_max(mut values: impl Iterator<Item = f64>) -> Option<f64> {
+        let mut v = values.next();
+        for candidate in values {
+            if candidate > v.unwrap() {
```

Comment:
This seems like a suboptimal implementation as that can easily produce all the panic code, unwinding code, etc.

Maybe this should be

```rust
let mut v = values.next()?;
for candidate in values {
    if candidate > v {
        v = candidate;
    }
}

Some(v)
```